### DISCUSSION TITLE
Possible function for pathway longitudinal analysis

### DISCUSSION THREAD

**Discussion by s6aamani** [david-barnett/microViz#67] [General]
Hi,
       I have projects where I typically generate pathways from 16s data. All of them are multiple time points and multiple groups. Could you suggest a function to do various linear mixed models considering the distribution of pathway abundance?
Thank you

**Comment by david-barnett** [comment]
Hi Aakash, 

I guess you're talking about the functional capacity inference using something like Picrust? microViz functionality is currently focused around taxonomic features (although you can actually put whatever abundance table you like in a phyloseq OTU table), and there's only experimental support for mixed models currently in tax_model.

was there a particular feature of microViz that you were interested in using for your aims?

**Reply by s6aamani** [reply]
Hi David,
     thank you for the quick reply. I liked ur microviz package. I am using it to build bar plots for pathways derived from picrust. I guess I can try to use the mixed model version of tax model for my analysis as well.
     To experiment this, I tried putting female*overweight in the tax_model function in this example https://david-barnett.github.io/microViz/reference/tax_model.html 
It seems to work fine. I would like your input on whether it is appropriate to use or not.
Thank you

**Reply by david-barnett** [reply]
Interaction terms like that will work fine. If you want mixed models you would need to change the type argument to lmer and currently install microViz from an alternative branch 
See https://github.com/david-barnett/microViz/issues/51

**Reply by s6aamani** [reply]
thank you for the feedback.
